quickstarts/angular1/01-login.md

```diff
@@ -0,0 +1,222 @@
+
+${snippet(meta.snippets.dependencies)}
+
+These may seem like a lot of dependencies, but each one has a very important function. Go through and figure out if each package works for your use case. Feel free to strip off the unnecessary packages.
+
+ - **lock** is the default authentication widget provided by Auth0. It is completely optional but I suggest you stick to it as an Auth0 newbie.
+ - **angular** is Angular's main library which you are building the application on.
+ - **angular-cookies** is Angular's wrapper for managing client cookies.
+ - **angular-route** is used to mange SPA routes in Angular application.
+ - **auth0-angular**: Auth0's SDK for Angular. Exposes most of the useful methods for authentication
+ - **angular-storage**: A `localStorage` and `sessionStorage` wrapper create with love by Auth0 team.
+ - **angular-jwt**: Angular service that makes using JWT easy in Angular apps
+
+ ::: panel-info Note
+ Some of the includes like angular-route and angular-storage will not be used in this step but will be useful in subsequent steps
+ :::
+
+Right after including the scripts, add a viewport to make the lock widget fit in to device widths:
+
+```html
+<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+```
+
+One more thing you can do in setting up is add a simple style sheet so the app does not look so boring:
+
+```html
+<link href="https://cdn.auth0.com/styleguide/4.4.24/index.css" rel="stylesheet" />
+```
+
+With all that, our entire `<head>` will look like this:
+
+${snippet(meta.snippets.head)}
+
+#### 2: Configure The Angular Application
+To use Auth0, your Angular app instance **MUST** depend on the Auth0 SDK:
+
+```javascript
+//app.js
+var app = angular.module('YOUR-APP-NAME', ['auth0']);
+```
+
+Next, go ahead and add the app name in your html:
+
+```html
+<html ng-app="YOUR-APP-NAME">
+```
+
+An Angular-Auth0 app requires basically 2 configurations:
+
+ - Auth0's credential config with `init()`
+ - Event listeners to handle authentication status
+
+Let's break it down.
+
+Angular's `config()` skeleton with required dependencies:
+```javascript
+app.config( function myAppConfig (authProvider) {
+  //authProvider init configuration
+})
+```
+
+The `authProvider` dependency is Auth0's API that exposes some methods including the `init()` method for for configuration:
+
+${snippet(meta.snippets.init)}
+
+Event listeners are available to handle different status of authentication. They are also configured in the `config()` method with `authProvider`:
+
+```javascript
+//Called when login is successful
+authProvider.on('loginSuccess', function(idToken) {
+  console.log("Login successful with token " + idToken);
+});
+
+//Called when login fails
+authProvider.on('loginFailure', function() {
+  alert("Error");
+});
+
+//Called when user is authenticated (Authentication flag is true)
+authProvider.on('authenticated', function() {
+  console.log("Authenticated");
```

Comment:
What is the difference between `loginSuccess` and `authenticated`? Do we need both?